nevergrad/functions/gym/gym_multi.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import numpy as np
+import os
+import gym
+
+if os.name != "nt":
+    import gym_anm  # pylint: disable=unused-import
+from nevergrad.parametrization import parameter
+from ..base import ExperimentFunction
+
+
+# Method for building a new list, for a future version of gym:
+#
+# GYM_ENV_NAMES = ["gym_anm:ANM6Easy-v0"]
+#
+# for e in gym.envs.registry.all():
+#     try:
+#         assert "Kelly" not in e.id
+#         env = gym.make(e.id)
+#         a1 = env.action_space.sample()
+#         a2 = env.action_space.sample()
+#         a3 = env.action_space.sample()
+#         a1 = a1 + a2 + a3
+#         if hasattr(a1, "size"):
+#             assert a1.size() < 15000
+#         GYM_ENV_NAMES.append(e.id)
+#     except:
+#         pass
+
+GYM_ENV_NAMES = [
```

Comment:
probably better to actually move them as class attributes, and remove the corresponding functions that you added

Reply:
It is now built on the fly. 
You still prefer to move then as class attributes ?   (can do that, just asking if the one-the-fly building changes something in your preference)

Reply:
Done anyway.